mist-lib/src/db/migrations/0006_genes_additional_indices.sql

```diff
@@ -0,0 +1,6 @@
+CREATE index ON genes(old_locus);
+CREATE INDEX trgm_idx_genes_product ON genes USING gin (product gin_trgm_ops);
+
+-- -- MIGRATION DOWN SQL
+-- DROP INDEX genes_old_locus_idx;
+-- DROP INDEX trgm_idx_genes_product;
```

Comment:
Lowercase all sql keywords.

Reply:
OK. I think one file instead of 0005_... and 0006_... is enough here. So I'll move the content of 0006_... to 0005_...

Reply:
OK.

I think one file instead of 0005_... and 0006_... files is enough. So I'll move the content of 0005_... to 0006_...